test_runner/src/main/kotlin/ftl/args/AndroidArgs.kt

```diff
@@ -151,5 +153,8 @@ val AndroidArgs.isSanityRobo
 val AndroidArgs.isGameLoop
     get() = if (type == null) false else (type == Type.GAMELOOP)
 
+val AndroidArgs.shardsFilePath
+    get() = Paths.get(localResultDir, resultsDir, ANDROID_SHARD_FILE).toAbsolutePath().toString()
```

Comment:
if `localResultsDir` is different than default, we should not add file to `resultsDir`. We have similar logic in the whole app